### PR TITLE
spread: use the official image for Ubuntu 20.10, no longer an unstable system

### PR DESCRIPTION
**Pull request by bboozzoo**
Groovy was released, update spread.yaml.

### CODE REVIEW

**Review by stolowski — APPROVED** [review]
Thanks!

**Review by mvo5 — APPROVED** [review]
Thanks!

**Comment by bboozzoo** [comment]
And we have a bunch of failures:
```
2020-11-04 11:37:34 Aborted tasks: 0
2020-11-04 11:37:34 Failed tasks: 5
    - google:ubuntu-20.10-64:tests/main/econnreset
    - google:ubuntu-20.10-64:tests/main/interfaces-fwupd-classic
    - google:ubuntu-20.10-64:tests/main/interfaces-packagekit-control
    - google:ubuntu-20.10-64:tests/main/snap-network-errors
    - google:ubuntu-20.10-64:tests/main/snap-run
```

File: spread.yaml [hunk 1]
```diff
@@ -102,6 +102,9 @@ backends:
                   secure-boot: true
                   # XXX: old name, remove once new spread is deployed
                   secureboot: true
+            - ubuntu-20.10-64:
+                  workers: 6
+                  image: ubuntu-20.10-64
```
Comment (sergiocazzolato) [inline spread.yaml:107]: This line is not needed

**Pull request merged**